README.md

```diff
@@ -66,7 +66,7 @@ It supports all their features, and many more.
 
 1) Install [Node.js](https://nodejs.org/en/download/) >= 12.0.0
```

Comment:
Good catch, updated. https://github.com/dr4fters/dr4ft/pull/1622/commits/3065591438092f44b2c7515979d0ee144e520366

I saw that [build was failing](https://github.com/dr4fters/dr4ft/runs/6683473557?check_suite_focus=true) on the `Run npm install` step due to this error:
```
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t git://github.com/dr4fters/utils.git
npm ERR! 
npm ERR! fatal: unable to connect to github.com:
npm ERR! github.com[0: 140.82.112.3]: errno=Connection timed out
```

Seems like it could have been a transient issue installing the https://github.com/dr4fters/utils dependency. The build is going again after pushing the above commit so we'll find out shortly.

Reply:
Doesn't appear to be a transient issue, though it does appear related to the `git+ssh://` protocol. Reading github issue https://github.com/actions/setup-node/issues/214 has been somewhat illuminating, but I implemented some suggested fixes from there (and force pushed them back out) to no avail. Seems to be an issue with older npm versions https://github.com/npm/cli/issues/2610.

Reply:
I can repro this on master (commit https://github.com/dr4fters/dr4ft/commit/0c02a922c420730d95a272a560a97b9c3274a69d), actually:
```bash
$ nvm use
Found '/home/keeler/g/dr4ft/.nvmrc' with version <v12>
Now using node v12.11.0 (npm v6.11.3)
$ npm install
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t git://github.com/dr4fters/utils.git
npm ERR! 
npm ERR! fatal: unable to connect to github.com:
npm ERR! github.com[0: 140.82.112.3]: errno=Connection timed out
npm ERR! 
npm ERR! 
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in: 
npm ERR!     /home/keeler/.npm/_logs/2022-06-02T04_44_49_034Z-debug.log
$ nvm use 14
Now using node v14.19.3 (npm v6.14.17)
$ npm install
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t git://github.com/dr4fters/utils.git
npm ERR! 
npm ERR! fatal: unable to connect to github.com:
npm ERR! github.com[0: 140.82.112.3]: errno=Connection timed out
npm ERR! 
npm ERR! 
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/keeler/.npm/_logs/2022-06-02T04_53_12_861Z-debug.log
```

I see the same failure on the [CI build for that commit](https://github.com/dr4fters/dr4ft/runs/6439216355?check_suite_focus=true) on master as well.

Reply:
It works if the dependency is specified with the `git+https` protocol instead of `git+ssh`. https://github.com/dr4fters/dr4ft/pull/1622/commits/5083990021e65111dcb9c3a43b4c7386e05248d7

It's hacky, but it uses sed to replace all instances of `git+ssh` with `git+https` in `package-lock.json` when using Node 14. If you try to set the `utils` dependency as `https://...` instead of `git://` in `package.json`, the npm CLI will override that and tell `package-lock.json` to use `git+ssh` anyway. The alternative would be to avoid installing the `utils` dependency through `package.json` either by adding that repo as a submodule or by simply copying the small amount of code there into this project. Not sure what other repos may or may not depend on `github.com/dr4fters/utils`.

Reply:
That is an interesting find and fix.
Also thanks for the comprehensive explanation! Appreciated 👍 

>Genuine question, is there a reason to continue to test for Node 14 support?

It's the maintenance LTS version and probably still in use I guess?